config/config.exs

```diff
@@ -136,6 +136,15 @@ config :archethic, Archethic.Networking.IPLookup.NATDiscovery,
 config :archethic, Archethic.Networking.IPLookup.RemoteDiscovery,
   provider: Archethic.Networking.IPLookup.RemoteDiscovery.IPIFY
 
+config :geolix,
+  databases: [
+    %{
+      id: :city,
+      adapter: Geolix.Adapter.MMDB2,
```

Comment:
This is really nice, I was not able to find the mamind db . Sam will be really happy : ) nice

Reply:
We will now have very less offset . We can now think about Network patches ?

Reply:
While it might improve location, network patch is something which might improve P2P calls, but not really priority for now.